src/building/bootstrapping.md

```diff
@@ -141,7 +141,8 @@ The convention `x.py` uses is that:
 - `x.py test --stage 0 library/std` means to run tests on the standard library
   without building `rustc` from source ('build with stage 0, then test the artifacts').
 - `x.py test --stage 0 src/test/ui` is not meaningful: it runs tests on the
-  _beta_ compiler and doesn't build `rustc` from source.
+  _beta_ compiler and doesn't build `rustc` from source. Use `test src/test/ui` instead, which builds from source.
+- `x.py test --stage 0 compiler/rustc` builds the compiler but runs no tests: it's running `cargo test -p rustc`, but cargo doesn't understand Rust's tests. You shouldn't need to use this, use `test` instead (without arguments).
```

Comment:
I think there actually are some unit tests in the compiler... but very few.
I agree with "do not usually use this"; I never ran this command.